applications/asset_tracker_v2/src/modules/data_module.c

```diff
@@ -371,6 +371,7 @@ static int config_settings_handler(const char *key, size_t len,
 			LOG_ERR("Failed to load configuration, error: %d", err);
 		} else {
 			LOG_DBG("Device configuration loaded from flash");
+			err = 0;
```

Comment:
Does `read_cb()` sometimes return a positive value > 0?

Reply:
yes it returns the number of bytes read 🙃

Reply:
of course, makes sense